src/main/java/seedu/address/model/food/Carbohydrate.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.food;
+
+public class Carbohydrate extends Macronutrient {
+
+    public Carbohydrate(int amount) {
+        super(amount, 4);
+    }
+
+    @Override
+    public String toString() {
+        return "MacronutrientType: Carbohydrate\n"
+                + "Amount: "
+                + this.amount + "\n"
+                + "Caloric Count: " + this.totalCalories + "\n";
+    }
+
+    // getter
+    public int getAmount() {
```

Comment:
Similar to the `toString` function, I think we can factor this out to the superclass. If we do that then we can also change all its `protected` access modifiers to the ideologically superior `private`.